### ISSUE TITLE
Errors upon attempting to install packages.

### ISSUE THREAD

**Issue by pnelligan** [wbond/package_control#1380]
Hello, and apologies in advance if there is an identical open issue.

I have issues installing packages with a new install of ST3 on OSX 10.13.6. I tried both dev build (3175) and latest build (3176), installed package control with 

```
import urllib.request,os,hashlib; h = '6f4c264a24d933ce70df5dedcf1dcaee' + 'ebe013ee18cced0ef93d5f746d80ef60'; pf = 'Package Control.sublime-package'; ipp = sublime.installed_packages_path(); urllib.request.install_opener( urllib.request.build_opener( urllib.request.ProxyHandler()) ); by = urllib.request.urlopen( 'http://packagecontrol.io/' + pf.replace(' ', '%20')).read(); dh = hashlib.sha256(by).hexdigest(); print('Error validating download (got %s instead of %s), please try manual install' % (dh, h)) if dh != h else open(os.path.join( ipp, pf), 'wb' ).write(by)
```

Here is the trace:
```
reloading plugin Package Control.1_reloader
reloading plugin Package Control.2_bootstrap
reloading plugin Package Control.Package Control
Package Control: Not running package cleanup since bootstrapping is not yet complete
reloading settings Packages/Package Control/Package Control.sublime-settings
reloading plugin 0_package_control_loader.00-package_control
reloading settings Packages/User/Package Control.sublime-settings
Package Control: Exported certificate: Common Name: Entrust.net Certification Authority (2048); Organizational Unit: (c) 1999 Entrust.net Limited, www.entrust.net/CPS_2048 incorp. by ref. (limits liab.); Organization: Entrust.net
Package Control: Exported certificate: Common Name: UTN-USERFirst-Object, Organizational Unit: http://www.usertrust.com, Organization: The USERTRUST Network, Locality: Salt Lake City, State/Province: UT, Country: US
Package Control: Exported certificate: Common Name: COMODO ECC Certification Authority, Organization: COMODO CA Limited, Locality: Salford, State/Province: Greater Manchester, Country: GB
Package Control: Exported certificate: Common Name: Buypass Class 3 Root CA, Organization: Buypass AS-983163327, Country: NO
Package Control: Exported certificate: Common Name: ComSign CA, Organization: ComSign, Country: IL
Package Control: Exported certificate: Organizational Unit: certSIGN ROOT CA, Organization: certSIGN, Country: RO
Package Control: Exported certificate: Email Address: info@e-szigno.hu, Common Name: Microsec e-Szigno Root CA 2009, Organization: Microsec Ltd., Locality: Budapest, Country: HU
Package Control: Exported certificate: Common Name: Swisscom Root CA 1, Organizational Unit: Digital Certificate Services, Organization: Swisscom, Country: ch
Package Control: Exported certificate: Common Name: Symantec Class 3 Public Primary Certification Authority - G6, Organizational Unit: Symantec Trust Network, Organization: Symantec Corporation, Country: US
Package Control: Exported certificate: Common Name: AffirmTrust Commercial, Organization: AffirmTrust, Country: US
Package Control: Exported certificate: Common Name: VeriSign Class 3 Public Primary Certification Authority - G5; Organizational Unit: (c) 2006 VeriSign, Inc. - For authorized use only, VeriSign Trust Network; Organization: VeriSign, Inc.; Country: US
Package Control: Exported certificate: Common Name: VeriSign Class 1 Public Primary Certification Authority - G3; Organizational Unit: (c) 1999 VeriSign, Inc. - For authorized use only, VeriSign Trust Network; Organization: VeriSign, Inc.; Country: US
Package Control: Exported certificate: Common Name: UCA Root, Organization: UniTrust, Country: CN
Package Control: Exported certificate: Email Address: acraiz@suscerte.gob.ve, Organizational Unit: Superintendencia de Servicios de Certificacion Electronica, Organization: Sistema Nacional de Certificacion Electronica, State/Province: Distrito Capital, Locality: Caracas, Country: VE, Common Name: Autoridad de Certificacion Raiz del Estado Venezolano
Package Control: Exported certificate: Common Name: VeriSign Class 3 Public Primary Certification Authority - G4; Organizational Unit: (c) 2007 VeriSign, Inc. - For authorized use only, VeriSign Trust Network; Organization: VeriSign, Inc.; Country: US
Package Control: Exported certificate: Common Name: thawte Primary Root CA - G3; Organizational Unit: (c) 2008 thawte, Inc. - For authorized use only, Certification Services Division; Organization: thawte, Inc.; Country: US
Package Control: Exported certificate: Common Name: OpenTrust Root CA G1, Organization: OpenTrust, Country: FR
Package Control: Exported certificate: Common Name: Echoworx Root CA2, Organizational Unit: Certification Services, Organization: Echoworx Corporation, Locality: Toronto, State/Province: Ontario, Country: CA
Exception in thread Thread-2:
Traceback (most recent call last):
  File "./python3.3/threading.py", line 901, in _bootstrap_inner
  File "/Users/paul/Library/Application Support/Sublime Text 3/Installed Packages/Package Control.sublime-package/package_control/commands/install_package_command.py", line 45, in run
    self.package_list = self.make_package_list(['upgrade', 'downgrade', 'reinstall', 'pull', 'none'])
  File "/Users/paul/Library/Application Support/Sublime Text 3/Installed Packages/Package Control.sublime-package/package_control/package_installer.py", line 58, in make_package_list
    packages = self.manager.list_available_packages()
  File "/Users/paul/Library/Application Support/Sublime Text 3/Installed Packages/Package Control.sublime-package/package_control/package_manager.py", line 694, in list_available_packages
    return self._list_available()[0]
  File "/Users/paul/Library/Application Support/Sublime Text 3/Installed Packages/Package Control.sublime-package/package_control/package_manager.py", line 554, in _list_available
    repositories = self.list_repositories()
  File "/Users/paul/Library/Application Support/Sublime Text 3/Installed Packages/Package Control.sublime-package/package_control/package_manager.py", line 453, in list_repositories
    channel_repositories = provider.get_repositories()
  File "/Users/paul/Library/Application Support/Sublime Text 3/Installed Packages/Package Control.sublime-package/package_control/providers/channel_provider.py", line 201, in get_repositories
    self.fetch()
  File "/Users/paul/Library/Application Support/Sublime Text 3/Installed Packages/Package Control.sublime-package/package_control/providers/channel_provider.py", line 87, in fetch
    channel_json = manager.fetch(self.channel, 'Error downloading channel.')
  File "/Users/paul/Library/Application Support/Sublime Text 3/Installed Packages/Package Control.sublime-package/package_control/download_manager.py", line 340, in fetch
    return self.downloader.download(url, error_message, timeout, 3, prefer_cached)
  File "/Users/paul/Library/Application Support/Sublime Text 3/Installed Packages/Package Control.sublime-package/package_control/downloaders/urllib_downloader.py", line 117, in download
    self.setup_opener(url, timeout)
  File "/Users/paul/Library/Application Support/Sublime Text 3/Installed Packages/Package Control.sublime-package/package_control/downloaders/urllib_downloader.py", line 317, in setup_opener
    bundle_path = get_ca_bundle_path(self.settings)
  File "/Users/paul/Library/Application Support/Sublime Text 3/Installed Packages/Package Control.sublime-package/package_control/ca_certs.py", line 31, in get_ca_bundle_path
    system_ca_bundle_path = get_system_ca_bundle_path(settings)
  File "/Users/paul/Library/Application Support/Sublime Text 3/Installed Packages/Package Control.sublime-package/package_control/ca_certs.py", line 166, in get_system_ca_bundle_path
    trust_list.get_path(ca_bundle_dir, hours_to_cache, cert_callback=print_cert_subject)
  File "/Users/paul/Library/Application Support/Sublime Text 3/Installed Packages/Package Control.sublime-package/package_control/deps/oscrypto/trust_list.py", line 122, in get_path
    for cert, trust_oids, reject_oids in extract_from_system(cert_callback):
  File "/Users/paul/Library/Application Support/Sublime Text 3/Installed Packages/Package Control.sublime-package/package_control/deps/oscrypto/_osx/trust_list.py", line 153, in extract_from_system
    _cert_callback(cert_callback, certificates[cert_hash], None)
  File "/Users/paul/Library/Application Support/Sublime Text 3/Installed Packages/Package Control.sublime-package/package_control/deps/oscrypto/_osx/trust_list.py", line 177, in _cert_callback
    callback(x509.Certificate.load(der_cert), reason)
  File "/Users/paul/Library/Application Support/Sublime Text 3/Installed Packages/Package Control.sublime-package/package_control/ca_certs.py", line 108, in print_cert_subject
    cert.subject.human_friendly
  File "/Users/paul/Library/Application Support/Sublime Text 3/Installed Packages/Package Control.sublime-package/package_control/deps/asn1crypto/x509.py", line 1042, in human_friendly
    native_value = self._recursive_humanize(value)
  File "/Users/paul/Library/Application Support/Sublime Text 3/Installed Packages/Package Control.sublime-package/package_control/deps/asn1crypto/x509.py", line 1071, in _recursive_humanize
    return value.native
  File "/Users/paul/Library/Application Support/Sublime Text 3/Installed Packages/Package Control.sublime-package/package_control/deps/asn1crypto/core.py", line 1100, in native
    return self.chosen.native
  File "/Users/paul/Library/Application Support/Sublime Text 3/Installed Packages/Package Control.sublime-package/package_control/deps/asn1crypto/core.py", line 1702, in native
    return self.__unicode__()
  File "/Users/paul/Library/Application Support/Sublime Text 3/Installed Packages/Package Control.sublime-package/package_control/deps/asn1crypto/core.py", line 1672, in __unicode__
    self._unicode = self._merge_chunks().decode(self._encoding)
  File "/Users/paul/Library/Application Support/Sublime Text 3/Installed Packages/Package Control.sublime-package/package_control/deps/asn1crypto/_teletex_codec.py", line 20, in decode
    return codecs.charmap_decode(input_, errors, DECODING_TABLE)
UnicodeDecodeError: 'charmap' codec can't decode byte 0x23 in position 32: character maps to <undefined>
```

**Comment by FichteFoll** [comment]
To me, it seems like you are seeing a nonstandard x.509 certificate in you ca store that pc (or the library it uses) cannot decode. I remember seeing a related issue so you might want to see whether that has a fix mentioned since I'm not really experienced in that area.

**Comment by deathaxe** [comment]
Should be solved by PC 3.4
